lib/good_migrations/configuration.rb

```diff
@@ -0,0 +1,29 @@
+module GoodMigrations
+  class Configuration
+    class << self
```

Comment:
I prefer to set up configuration objects as Structs or POROs and hold their instance in a memoized class method so that they can be tested better in isolation and reset to defaults. [Here's an example](https://github.com/testdouble/test_data/blob/main/lib/test_data/config.rb)